### PR TITLE
removing unecessary inclusion of the default recipe

### PR DESCRIPTION
**Pull request by seperman**
This also resolved the ticket: Manually deleted the conf files. Now getting: Cannot find a resource matching service[redis6379](did you define it first?) #170

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs/Configures redis'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.2.4'
+version          '2.2.5'
```
Comment (majormoses) [inline metadata.rb]: please do not include the version number when submitting changes there are several reasons not to:
- the maintainer could disagree with you on impact
- PRs are not a FIFO queue so other PRs merging with unrelated changes now forces a rebase in order to proceed (like in this case)
- some projects lump multiple changes into a single release (not that I am a fan on this)

**Comment by Xorima** [comment]
Hi
We have just adopted this cookbook and are aiming to rebaseline it, as part of this we are closing down all prs older than 1 year.
If you still need this change please open a new PR and we will assess this as part of our rebaseline
Thanks for your understanding
Sous-Chefs
